### PR TITLE
Team meeting minutes

### LINKED ISSUE #133: Team Meeting Minutes

### PR DESCRIPTION
**Pull request by edouarde1**
closes #133

### CODE REVIEW

File: docs/meetings/team/2022_11_22.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+1. Edouard
+2. Matt
+3. Veronica
+4. Logan
+
+***Attendence Comments***
+
+## 🧸 Check-In
+
+- "I only know 25 letters of the alphabet I don't y"
```
Comment (LoganParker) [inline docs/meetings/team/2022_11_22.md]:
```suggestion
- "I only know 25 letters of the alphabet I don't know y"
```

**Review by LoganParker — APPROVED** [review]
All looks great! thanks for taking these

File: docs/meetings/team/2022_11_22.md (hunk 1 above)
Comment (vnajack) [reply docs/meetings/team/2022_11_22.md]: somehow it doesn't have the same effect when written compared to spoken 😆

File: docs/meetings/team/2022_11_22.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+- 1) Kukeralla User Type Text, Download File
+- 2) Place in directory
+- 3) Run process
+- 4) Output Video
+  
+TASK: FFMPEG
+    - Logn
+
+ TASK: Error handling
```
Comment (vnajack) [inline docs/meetings/team/2022_11_22.md]:
```suggestion
TASK: Error handling
```

**Review by vnajack — APPROVED** [review]
Thank you for taking the minutes! Looks like we're in a good place this week 😄

**Pull request merged**